food-truck-frontend/src/components/search/SearchRow.tsx

```diff
@@ -0,0 +1,35 @@
+import React from 'react';
+import {Divider, ListItem, ListItemText} from "@material-ui/core";
+import {useRouter} from "next/router";
+
+type TruckProp = {
+    truck : {
+        id: number,
+        name: string,
+        description: string
+    }
+}
+
+function SearchTruckRow(truck: TruckProp){
+    const router = useRouter();
+
+    if(truck.truck.id == -1){
+        return(
+            <div>
+
+            </div>
+        )
+    }
+
+    return(
+        <div>
+            <Divider/>
+            <ListItem button onClick={() => {router.push(`/truck/${truck.truck.id}`)}}>
+                <ListItemText primary={truck.truck.name}/>
+                <ListItemText secondary={truck.truck.description ? truck.truck.description : "No Description"}/>
```

Comment:
Maybe adding a button to visit the truck's page would be nice. Not needed now, though.